### PR TITLE
Extend SparklyCatalog to work with database properties

### PR DESCRIPTION
**Pull request by drudim**
New methods:
- `spark.catalog_ext.set_database_property`
- `spark.catalog_ext.get_database_property`
- `spark.catalog_ext.get_database_properties`

### CODE REVIEW

File: sparkly/catalog.py [hunk 1]
```diff
@@ -203,6 +205,71 @@ def set_table_property(self, table_name, property_name, value):
...
+
+    def set_database_property(self, db_name, property_name, value):
+        """Set value for database property.
+
+        Args:
+            db_name (str): A database name.
+            property_name (str): A property name to set value for.
+            value (Any): Will be automatically casted to string.
+        """
+        property_name_blacklist = {',', '(', ')'}
```
Comment (skliarpawlo) [inline sparkly/catalog.py]: should this be a class level constants to be patchable if needed

Comment (drudim) [reply sparkly/catalog.py]: not sure, it doesn't make sense to change them without changing `read_db_properties_format`

**Comment by skliarpawlo** [comment]
Btw, docs are missing

**Comment by drudim** [comment]
Don't we have api docs for this?

**Comment by skliarpawlo** [comment]
I think so https://sparkly.readthedocs.io/en/latest/catalog.html#table-properties-management

**Pull request merged**